### PR TITLE
Add nav and footer

### PR DESCRIPTION
**Pull request by CjMoore**
Adds nav and footer partials in shared directory in views. Very basic, does not have links, will add as we add relative functionality.

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -2,13 +2,18 @@
...
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.5/js/materialize.min.js"></script>
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.5/css/materialize.min.css" rel="stylesheet"/>
     <%= csrf_meta_tags %>
 
     <%= stylesheet_link_tag    'application', media: 'all' %>
     <%= javascript_include_tag 'application' %>
   </head>
 
   <body>
+    <%= render partial: 'shared/nav' %>
```
Comment (cmacaulay) [inline app/views/layouts/application.html.erb:15]: 👍 partials 🎉

File: app/views/shared/_footer.html.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<footer class="page-footer orange darken-4">
+  <div class="footer-copyright">
+    <div class="container">
+      <p>
+        93 Diagon Alley, London, England
```
Comment (cmacaulay) [inline app/views/shared/_footer.html.erb:5]: Dream driven development idea: mapping this address. 😄

File: app/views/shared/_nav.html.erb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<nav>
+   <div class="nav-wrapper orange darken-3">
```
Comment (cmacaulay) [inline app/views/shared/_nav.html.erb:2]: @CjMoore I'm going to start an application style sheet with just a skeleton and the start of a color glossary. There are some cool color palette generators, I can use orange darken-3 as a launching off point and share some options with the group? Let me know if that sounds good.

File: spec/features/nav_footer/user_sees_nav_and_footer_spec.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  scenario "see nav at root" do
+    visit root_path
+
+    expect(page).to have_content("Weasleys' Wizard Wheezes")
+  end
+
+  scenario "see footer at root" do
+    visit root_path
+
+    expect(page).to have_content("93 Diagon Alley, London, England")
```
[the remark this answers is not in the document]
Comment (cmacaulay) [reply spec/features/nav_footer/user_sees_nav_and_footer_spec.rb:13]: 👎 hound!

File: spec/models/item_spec.rb [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    end
+
+    context "is invalid" do
+      it "without a title" do
+        item = Item.new( description: "a rocket from the Explosive Enterprises line by Weasleys' Wizard Wheezes", price: 10.99 )
+
+        expect(item).to be_invalid
+      end
+
+      it "without description" do
```
[the remark this answers is not in the document]
Comment (cmacaulay) [reply spec/models/item_spec.rb:20]: houndin' us!

**Review by cmacaulay — APPROVED** [review]
👍 Looks good to me! @meyerhoferc @valerietrudell ready for one of you to review.

File: app/views/shared/_nav.html.erb (hunk 3 above)
Comment (CjMoore) [reply app/views/shared/_nav.html.erb:2]: Sounds like a plan.

**Comment by meyerhoferc** [comment]
looks good to me. let's discuss tomorrow how to handle these hound alarms

**Comment by cmacaulay** [comment]
Word. Charlotte mentioned being able to configure it to not include certain things.

**Pull request merged**